releases/Dockerfile.linux

```diff
@@ -10,14 +10,14 @@ LABEL maintainer="Microsoft" \
       org.label-schema.name="Azure Kubernetes Engine (aks-engine)" \
       org.label-schema.version=$AKSENGINE_VERSION \
       org.label-schema.license="MIT" \
-      org.label-schema.description="The Azure Kubernetes Engine (aks-engine) generates ARM (Azure Resource Manager) templates for Kubernetes clusters on Microsoft Azure with your choice of DCOS, Kubernetes, or Swarm orchestrators." \
+      org.label-schema.description="The Azure Kubernetes Engine (aks-engine) generates ARM (Azure Resource Manager) templates for Kubernetes clusters on Microsoft Azure." \

```

Comment:
Thoughts on adding a label for maintainer email?

Reply:
We could... Maybe change the [maintainer label](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated) from "Microsoft" to an email address if we want to do that. I didn't see anything standardized in [label-schema](http://label-schema.org/rc1/) for maintainer email.

Reply:
Sure. For later then!